unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarModifierAreaHUD/Scripts/AvatarModifierAreaFeedbackController.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using UnityEngine;
+
+namespace DCL.AvatarModifierAreaFeedback
+{
+    public class AvatarModifierAreaFeedbackController : IHUD
+    {
+        internal AvatarModifierAreaFeedbackView view;
+        private BaseVariable<int> InAvatarModifierAreaForSelfCounter => DataStore.i.HUDs.inAvatarModifierAreaForSelfCounter;
+        private BaseCollection<string> InAvatarModifierAreaWarningDescription => DataStore.i.HUDs.inAvatarModifierAreaWarningDescription;
+        
+        public AvatarModifierAreaFeedbackController()
+        {
+            view = AvatarModifierAreaFeedbackView.Create();
```

Comment:
An interface should be extracted from `AvatarModofierAreaFeedbackView` and used instead, also I would pass this `view` in the constructor to ease the tests.

Reply:
Done!